cypress/integration/page.spec.js

```diff
@@ -1036,7 +1114,7 @@
       page.spyOnCommentsPost();
 
       newCommentForm.nameInput().type('Some name');
-      newCommentForm.bodyArea().type('Some comment body')
+      newCommentForm.bodyArea().type('Some comment body');
```

Comment:
There is a missing semicolon at the end of the `cy.type` statement. Ensure that each statement is properly terminated to avoid syntax errors.